plugin/src/App/getTextBounds.lua

```diff
@@ -0,0 +1,26 @@
+local TextService = game:GetService("TextService")
+
+local Rojo = script:FindFirstAncestor("Rojo")
+local Packages = Rojo.Packages
+
+local Log = require(Packages.Log)
+
+local params = Instance.new("GetTextBoundsParams")
+
+local function getTextBounds(text: string, font: Font, textSize: number, width: number, richText: boolean?): Vector2
+	params.Text = text
```

Comment:
This property set should probably have a check for text with length >= 200000, right now it's possible for it to brick the entire Roblox UI